test/new-e2e/tests/windows/install-test/installtester.go

```diff
@@ -368,6 +368,12 @@ func (t *Tester) testCurrentVersionExpectations(tt *testing.T) {
 	require.NoError(tt, err)
 	serviceTester.TestInstall(tt)
 
+	tt.Run("user rights", func(tt *testing.T) {
+		AssertUserRights(tt, t.host,
+			windows.MakeDownLevelLogonName(t.expectedUserDomain, t.expectedUserName),
+		)
+	})
+
```

Comment:
Suggest in future PR move user checks is a separate block (e.g. https://github.com/DataDog/datadog-agent/pull/23419/files#diff-cf464d0b98afe8e5a03a4033f8584c6954d2eef118f6e471390c969b63436624R360-R376)